src/app/public/src/modules/wrapper/wrapper.component.scss

```diff
@@ -1,5 +1,15 @@
 @import '../../styles/_variables';
 
+/*
+* We add this overflow-y so that when switching between screens with and without a vertical
+* scroll bar the page doesn't "jump" 16px back and forth.
+* https://stackoverflow.com/a/7607206
```

Comment:
"Worth mentioning that this permanently shows a vertical scrollbar. May not always be acceptable." :(